shuffle/cli/src/deploy.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Context, Result};
+use anyhow::{bail, format_err};
+use diem_config::config::NodeConfig;
```

Comment:
Where is diem_config, diem_crypto, and diem_sdk in the repo?

Reply:
Ok found that diem_config gets this information from Diem/config/src/config and the rest follow the same folder structure. So to get access to something like NodeConfig, do we just need the outer most folder (diem_config) and the folder with the src in it (config) to get access to NodeConfig?

Reply:
all the use modules are defined in the Cargo.toml file within each directory, which is how you do imports of other modules in rust

Reply:
After defining the path in cargo.toml, you can run cargo build to do the actual import and then do `use projectname` to declare the modules in your file and use them